csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -908,7 +908,10 @@ public async Task SetPublishedEndpointsAsync(IEnumerable<Endpoint> newEndpoints)
                 IObject? servant = Find(current.Identity, current.Facet);
                 if (servant == null)
                 {
-                    throw new ObjectNotExistException(current.Identity, current.Facet, current.Operation);
+                    throw new ObjectNotExistException(current.Identity,
+                                                      current.Facet,
+                                                      current.Operation,
+                                                      RetryPolicy.OtherReplica);
```

Comment:
Wasn't the idea to use "OtherReplica" only if the OA replica group ID was configured?

Reply:
fixed